<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSettingsContainer</class>
 <widget class="QWidget" name="FormSettingsContainer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings Manager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="pushButtonAddNew">
     <property name="text">
      <string>Add</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxAddingOptions">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <property name="title">
      <string>Adding options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditName">
          <property name="text">
           <string>name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Project name:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Short description:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEditDescription">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButtonConfirm">
          <property name="text">
           <string>Confirm</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButtonCancel">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Settings list:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QScrollArea" name="scrollAreaSettingsContainer">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>429</width>
           <height>196</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayoutSettingsList">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonAddNew</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBoxAddingOptions</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>groupBoxAddingOptions</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>16</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonConfirm</sender>
   <signal>clicked()</signal>
   <receiver>groupBoxAddingOptions</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonAddNew</sender>
   <signal>clicked()</signal>
   <receiver>pushButtonAddNew</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>pushButtonAddNew</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonConfirm</sender>
   <signal>clicked()</signal>
   <receiver>pushButtonAddNew</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
